bindings/protocols/coap/index.html

```diff
@@ -53,7 +53,27 @@ <h2>Introduction</h2>
     </section>
     <section id="overview">
         <h2>Protocol Overview</h2>
-        <p class="ednote">TODO: Provide an overview of the protocol and the protocol binding.</p>
+        <p>
+            In contrast to HTTP, where header fields all follow the same key-value based structure,
+            CoAP defines specific formats for its options, which all have to be registered with IANA,
+            being the primary way for adding new features to the protocol.
+        </p>
+        <p>
+            Due to the highly standardized nature of CoAP options, mapping them to a vocabulary
+            of <em>features</em> makes more sense than offering a generic format for defining them
+            within a TD form, also considering that there options with binary, numeric or
+            string-based values (see sections 3.1 and 3.2 of [[RFC7252]] for more details).
+        </p>
+        <p>
+            The Table below gives an overview of how the features offered by the existing CoAP RFCs
```

Comment:
```suggestion
            The table below gives an overview of how the features offered by the existing CoAP RFCs
```

Reply:
Thank you for the review and feedback :)